apps/aecore/src/aec_conductor.erl

```diff
@@ -146,26 +145,12 @@ get_mining_workers() ->
 %%%===================================================================
 %%% State trees API
 
-%% TODO: Added for backwards compability.
--spec get_miner_account_balance() ->   {'ok', integer()}
-                                     | {'error', 'account_not_found'}.
-get_miner_account_balance() ->
-    {ok, Pubkey} = aec_keys:pubkey(),
+-spec get_account(pubkey()) -> {ok, account()} | {error, notfound}.
+get_account(Pubkey) ->
```

Comment:
This copies the full state to the calling process. Please make it a specific call instead.

I'm allowing this in this PR, though. You can fix that in the next stage if you please.

Reply:
Yes, coding right now, I will include in next stage.